db/migrate/20220624125915_create_favorites.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFavorites < ActiveRecord::Migration[7.0]
+  def change
+    create_table :favorites do |t|
+      t.integer :book_id
```

Comment:
same
`t.references :book, index: true`

it will automatically handle all the index/foreign key magic for you